src/platform_impl/linux/event_loop.rs

```diff
@@ -563,9 +601,9 @@ impl<T: 'static> EventLoop<T> {
 
             let tx_clone = event_tx.clone();
             let keyboard_handler = Rc::new(move |event_key: EventKey, element_state| {
-              // if we have a modifier lets send it
               let mut mods = keyboard::get_modifiers(event_key.clone());
-              if !mods.is_empty() {
```

Comment:
Why was this removed? Just making sure it isn't a typo.

Reply:
This is replaced by `if event_key.is_modifier()`. So event_key with modifier like `<Shift>T` can get the modifier state in the event.